services/hanke-service/src/test/kotlin/fi/hel/haitaton/hanke/logging/DisclosureLogServiceTest.kt

```diff
@@ -64,24 +70,26 @@ internal class DisclosureLogServiceTest {
         confirmVerified(auditLogService)
     }
 
-    @Test
-    fun `saveDisclosureLogsForProfiili saves audit log of the entire info`() {
-        val info =
-            CollectionNode(
-                "user",
-                listOf(
-                    StringNode("id", "4f15afe1-51dc-4015-bb66-3a536295abea"),
-                    StringNode("etunimi", TEPPO),
-                    StringNode("sukunimi", TESTIHENKILO),
-                    StringNode("sahkoposti", TEPPO_EMAIL),
-                    StringNode("puhelin", TEPPO_PHONE),
+    @Nested
```

Comment:
I think refactoring the tests to use nested test classes could had been done in a separate PR. This one could've had the new tests in an inner class but left the other unchanged. As-is it's really hard to see what has changed here.

Probably not worth it to undo the changes now, though.

Reply:
Reverted changes to other tests.